### ISSUE TITLE
Change behaviour of left-side navigation

### ISSUE THREAD

**Issue by davidsneighbour** [gethugothemes/gethugothemes-documentation-site#9]
I don't understand how the left side navigation is built (and I did not check how it's done ;)

What I am trying to achieve is a navigation that shows everything at the current level and below. If for instance the site structure is as follows:

```
content
|- topic 1
   |- subtopic 1
   |- subtopic 2
|- topic 2
   |- subtopic 1
   |- subtopic 2
```

I would want the navigation when visiting `/content/` on the resulting site to show the topics of topic 1 and the subsequent subtopic 1 and 2, not the topic 2 that is "parallel" to topic 1. 

Is it possible to change the theme to do this? 

Long story short: If I move the theme documentations into a subfolder themes then all themes show all theme-doc-navigations, instead of their own. 

We need a more flexible structure.

for instance:

```
content
|- guide
   |- guide topic 1
   |- guide topic 2
|- how tos
   |- topic 1
   |- topic 2
```

without showing nav content for the guide in the how to section.

Right now it just seems that the TOC is using ALL available content files... let's not ;)
